src/app/providers/CacheDetailProvider.tsx

```diff
@@ -40,6 +40,13 @@ const CacheDetailProvider = ({ children }) => {
     }
   };
 
+  const isEncodingAvailable = (): boolean => {
```

Comment:
export this to the file 

the Empty encoding is an encoding type enum. use ||, not and
can be inlined in a single like

```
return cache.encoding.key  ... ||  ... 
```